petastorm/spark/spark_dataset_converter.py

```diff
@@ -0,0 +1,343 @@
+#  Copyright (c) 2020 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atexit
+import shutil
+import threading
+import datetime
+import uuid
+import logging
+import warnings
+
+from pyarrow import LocalFileSystem
+from pyspark.sql.session import SparkSession
+from six.moves.urllib.parse import urlparse
+
+from petastorm import make_batch_reader
+from petastorm.fs_utils import FilesystemResolver
+
+DEFAULT_ROW_GROUP_SIZE_BYTES = 32 * 1024 * 1024
+
+
+def _get_spark_session():
+    return SparkSession.builder.getOrCreate()
+
+
+_parent_cache_dir_url = None
+
+
+def _get_parent_cache_dir_url():
+    """
+    Get parent cache dir url from `petastorm.spark.converter.parentCacheDirUrl`
```

Comment:
Define it as a constant, e.g., `SparkDatasetConverter.PARENT_CACHE_DIR_URL_CONF`.